addons/api/mirage/generated/factories/alias.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { Factory } from 'miragejs';
+import { faker } from '@faker-js/faker';
+import { TYPE_ALIAS_TARGET } from 'api/models/alias';
+
+/**
+ * GeneratedAliasModelFactory
+ */
+export default Factory.extend({
```

Comment:
I think we are missing `destination_id` field?

Reply:
added!